config/error.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2020 Layer5, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import (
+	"fmt"
+
+	"github.com/layer5io/gokit/errors"
```

Comment:
`gokit` may not in stable?

Reply:
Yes, it might not be. However, it is versioned and I think it is good to use it, then gain experience with it.
The current repository is most probably not stable yet either.

Just noted that I need to a tag to this repository here too, I suggest v0.1.0-alpha.1

Reply:
It's there some reason we must needs to use `github.com/layer5io/gokit/errors` repo? Is there any plan?

Reply:
The reason was that gokit is used in the kuma adapter, and this code is based on kuma adapter as well.

Are you referring to `errors` in particular, or `gokit`?

I agree we should discuss the plan. There is some discussion here https://docs.google.com/document/d/1b8JAMzr3Rntu7CudRaYv6r6ccACJONAB5t7ISCaPNuA/edit#heading=h.blih70a9hxp and we could also discuss in the issue you created https://github.com/layer5io/gokit/issues/1

Reply:
Ok, No problem.